pkg/config/config.go

```diff
@@ -129,6 +133,10 @@ func writeDefaultConfig(f *os.File) error {
 # This is the default configuration file. Available options are provided, but disabled.
 # Delete the leading hash mark to enable an item.
 
+# Uncomment the following line to make line numbers in (list) command show bright yellow
+# rather than dark blue.. much better for old eyes in a black terminal.
```

Comment:
remove `. much better for old eyes in a black terminal.`, too much editorializing.